src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
```

Comment:
chyba Ci IDE wygenerowało => zbędne – taki komentarz nic nie wnosi poza to, co jest jasne